storybook/src/components/FieldSet/FieldSet.stories.tsx

```diff
@@ -93,15 +108,54 @@ export const WithDescription: Story = {
   ),
 }
 
+export const WithNotRequiredLabel: Story = {
```

Comment:
This currently doesn't show the not required label. I'm not sure we really need this story, the not required label part works the same for all fieldsets. Showing it once is enough imo